action.yml

```diff
@@ -171,29 +140,57 @@ runs:
       env:
         GITHUB_TOKEN: ${{ inputs.pkg-token }}
         SONAR_TOKEN: ${{ inputs.sonar-token }}
-      run: ./gradlew sonarqube --info
       shell: bash
+      run: ./gradlew sonarqube --info
+
+
 
     - name: Build - Docker Image - echo
-      run: echo -e "🔔       D O C K E R      B U I L D"
       shell: bash
+      run: echo -e "🔔       D O C K E R      B U I L D"
 
     - name: Build - Docker Image
+      shell: bash
```

Comment:
why are we adding these shell: bash?

Reply:
Did not add any, just moved these lines above the run command.